platform/core/src/runs/TransactingMutationListener.js

```diff
@@ -38,9 +44,28 @@ define([], function () {
                     transactionService.startTransaction();
                 }
 
+                if (!MUTATION_TRACKER.has(domainObject)) {
+                    MUTATION_TRACKER.set(domainObject, domainObject
+                        .getCapability('mutation')
+                        .listen(function () {})
```

Comment:
> Transacting mutation listener tracks mutation for the duration of the transaction and places mutated object in cache.

This is where I'd expect to see that happen, but it looks like a no-op. 

I'm not sure why this would be necessary in any event, since any changes to any `domainObject` should be available on the `mutationTopic`